Deck_Shuffler/Deck_Shuffler/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Deck EddsDeck = new Deck();
+
+            Deck ShuffledDeck = Shuffler.Shuffle(EddsDeck);
```

Comment:
Lovely and readable Eddie. Well done!

Reply:
Thanks bro. I thought you might like that!